modules/vocabularies/src/main/frontend/src/vocabularyTable.jsx

```diff
@@ -87,55 +63,81 @@ export default function VocabularyTable(props) {
-                  );
-                }
-              })}
-            </TableBody>
-
-          </Table>
-        </Paper>
+        <MaterialTable
+            title=""
+            columns={[
+              { title: 'Id',
+                cellStyle: {
+                  width: '10%',
+                  whiteSpace: "pre",
+                  fontWeight: '400'
+                },
+                field: 'acronym'
+              },
+              { title: 'Name',
+                cellStyle: {
+                  width: "33%",
+                  whiteSpace: "pre",
+                  fontWeight: '400'
+                },
+                field: 'name'
+              },
+              { title: 'Version',
+                cellStyle: {
+                  width: "20%",
+                  whiteSpace: "pre",
+                  fontWeight: '400'
+                },
+                field: 'version'
+              },
+              {
+                title: type === "local" ? "Installation Date" : "Release Date",
+                cellStyle: {
+                  width: "14%",
+                  whiteSpace: "nowrap",
+                  fontWeight: '400'
+                },
+                render: rowData => (new Date(type === "local" ? rowData.installed : rowData.released)).toString().substring(4,15)
+              },
+              { title: "Actions",
+                cellStyle: {
+                  width: "23%",
+                  whiteSpace: "pre",
+                  textAlign: "right"
+                },
+                render: rowData => <VocabularyActions
+                                     type={type}
+                                     vocabulary={rowData}
+                                     updateLocalList={props.updateLocalList}
+                                     // If filterTable is True, then check if the acronym is of a vocabulary to be displayed
+                                     // If filterTable is False, then don't hide anything
+                                     hidden={filterTable && !acronymList.includes(rowData.acronym)}
+                                     initPhase={props.acronymPhaseObject[rowData.acronym] || Phase["Not Installed"]}
+                                     setPhase={(phase) => props.setPhase(rowData.acronym, phase)}
+                                     addSetter={(setFunction) => props.addSetter(rowData.acronym, setFunction, type)}
+                                   />
+              }
+            ]}
+            data={vocabList}
+            options={{
+              search: true,
+              exportButton: true,
+              emptyRowsWhenPaging: false,
+              addRowPosition: 'first',
+              pageSize: rowCount,
+              headerStyle: { backgroundColor: theme.palette.grey['200'],
+                             textAlign: 'center'
+                           },
+            }}
+            localization={{
+              toolbar : { searchTooltip: "Filter",
```

Comment:
I feel like this should be more explicit. How about:
```suggestion
              toolbar : { searchTooltip: "Filter by vocabulary id or name ",
```